README.md

```diff
@@ -3,3 +3,5 @@ Practicing git.
 
 Lesson 1: doing PR
 
+Lesson 2: explaining review to ....
+
```

Comment:
This can include a datestamp when lesson took place, for example:

**April 26th, 2021**